coconut/configuration/configuration.go

```diff
@@ -155,34 +147,29 @@ func List(cfg *configuration.ConsulSource, cmd *cobra.Command, args []string, o
 	keyPrefix := componentsPath
 	useTimestamp := false
 	if len(args) > 1 {
-		err = errors.New(fmt.Sprintf("Command requires maximum 1 arg but received %d", len(args)))
-		return err , invalidArgs
+		return errors.New(fmt.Sprintf("Command requires maximum 1 arg but received %d", len(args))) , invalidArgs
 	} else {
 		useTimestamp, err = cmd.Flags().GetBool("timestamp")
 		if err != nil {
-			err = errors.New(fmt.Sprintf("Flag `-t / --timestamp` could not be identified"))
-			return err, invalidArgs
+			return err,  invalidArgs
 		}
 		if len(args) == 1 {
-			if !IsInputSingleValidWord(args[0]) {
-				err = errors.New(fmt.Sprintf("Requested component name cannot contain character `/` or `@`"))
-				return err, invalidArgs
+			if !isInputSingleValidWord(args[0]) {
+				return  errors.New(fmt.Sprintf(invalidArgsErrMsg)), invalidArgs
```

Comment:
This may simply be `errors.New(invalidArgsErrMsg)`. [https://golang.org/pkg/errors/#New](https://golang.org/pkg/errors/#New)
`fmt.Sprintf() ` is not necessary if you don't have some fancy arguments you want to handle.

Reply:
Did not know. Thank you 👍

Reply:
There's also `fmt.Errorf` that combines the two for when you **do** need to handle fancy arguments ;)